<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomWidget</class>
 <widget class="QWidget" name="ZoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>98</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="zoomInButton">
     <property name="text">
      <string>+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="zoomSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomOutButton">
     <property name="text">
      <string>-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="zoomSpinBox">
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fitButton">
     <property name="text">
      <string>Fit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
